tests/scalapack/scalapack_05.mpirun=1.output

```diff
@@ -1,36 +1,36 @@
 32 32 1 1
-0.410521 0.471758
```

Comment:
does this output pass the test? I would expect the first number to have more digits